src/cms/templates/dashboard/admin_dashboard.html

```diff
@@ -24,7 +24,7 @@ <h2 class="font-bold">{% trans 'At a glance' %}</h2>
             </button>
             <div class="collapsible-content flex flex-col active pb-4">
                 <div class="mb-1 text-s text-gray-600">{% trans 'The five most recent entries are listed.' %}</div>
-                <table class="w-full rounded border border-solid border-gray-200 shadow bg-white">
+                <table class="w-full rounded border border-solid border-gray-200 shadow bg-white" data-enable-row-checkbox-toggle>
```

Comment:
```suggestion
                <table class="w-full rounded border border-solid border-gray-200 shadow bg-white">
```
Cool possibility to generalize the toggling of row checkboxes! But the feedback in the admin dashboard doesn't have checkboxes, so we don't need it there I think.